src/open_inwoner/openzaak/admin.py

```diff
@@ -13,15 +13,11 @@
 from django.utils.html import format_html, format_html_join
 from django.utils.translation import gettext_lazy as _, ngettext
 
-from import_export.admin import ImportExportMixin
 from privates.storages import PrivateMediaFileSystemStorage
 from solo.admin import SingletonModelAdmin
 
 from open_inwoner.ckeditor5.widgets import CKEditorWidget
-from open_inwoner.openzaak.import_export import (
-    CatalogusConfigExport,
-    CatalogusConfigImport,
-)
+from open_inwoner.openzaak.import_export import CatalogusConfigImport, ZGWConfigExport
```

Comment:
I guess we could propagate the rename to the import container as well:

```suggestion
from open_inwoner.openzaak.import_export import ZGWConfigImport, ZGWConfigExport
```